src/scout_map.py

```diff
@@ -49,7 +49,8 @@ def __init__(self, census_csv_path):
         self.OUTPUT = self.settings["Output folder"]
 
         # Facilitates logging
-        self.logger = log_util.create_logger(__name__,)
+        self.logging = log_util.LogUtil(__name__, )
+        self.logger = self.logging.get_logger()
```

Comment:
Why do we get another logger, given that the LogUtil class already has a logger in it? I think it's confusing to have self.logging and self.logger - I think I would rather that the log_util class had debug, info functions etc. - it might be wise for the LogUtil class to be a child of the logger object, but not sure ...